platforms/shared/scripts/dev_prerequisites_setup.bat

```diff
@@ -0,0 +1,212 @@
+git --version
+set GITVAR=%errorlevel%
+if %GITVAR%==9009 (
+echo Starting git download v2.26.0 64 bit
+powershell -Command "Invoke-WebRequest https://github.com/git-for-windows/git/releases/download/v2.26.0.windows.1/Git-2.26.0-64-bit.exe -OutFile gitsetup.exe"
+if %ERRORLEVEL%==1 (
+echo Bad internet/url. Exiting...
+PAUSE
+EXIT
+)
+echo Git download successful
+echo Installing git. Press 'Next' at each prompt so that it gets installed with default settings
+START /WAIT gitsetup.exe
+echo git successfully installed
+) else (
+echo git already installed
+)
+REM ############################################################################################################################################################
+REM ############################################################################################################################################################
+
+REM Configuring git username and useremail
+echo Please create a github account and continue
+PAUSE
+
+set /p GIT_USERNAME=Enter git username: 
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "git config user.name" > CHECK_USERNAME.txt
+set /P CHECK_USERNAME=<CHECK_USERNAME.txt
+if %CHECK_USERNAME% NEQ %GIT_USERNAME% ( 
+echo Updating username
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "git config --global user.name %GIT_USERNAME%"
+)
+del CHECK_USERNAME.txt
+
+set /p GIT_USEREMAIL=Enter user email: 
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "git config user.email" > CHECK_USEREMAIL.txt
+set /P CHECK_USEREMAIL=<CHECK_USEREMAIL.txt
+if %CHECK_USEREMAIL% NEQ %GIT_USEREMAIL% (
+echo Updating useremail
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "git config --global user.email %GIT_USEREMAIL%"
+)
+del CHECK_USEREMAIL.txt
+
+echo Git configured successfully
+
+echo Configuring git so that EOLs are not updated to Windows CRLF
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "git config --global core.autocrlf false"
+
+REM ############################################################################################################################################################
+REM ############################################################################################################################################################
+
+REM Setting up of forked repo on local machine and checkout to a given branch ( defaulted to develop)
+echo Please fork the blockchain-automation-framework repository from browser.
+PAUSE
+
+"C:\Program Files\Git\bin\sh.exe" --login -i -c 'ssh-keygen -q -N "" -f ~/.ssh/gitops'
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "eval $(ssh-agent)"
+
+md project
+chdir project
+set /p REPO_URL=Enter your forked repo clone url (HTTPS url): 
+git clone %REPO_URL%
+set /p REPO_BRANCH=Enter branch(default is develop): 
+chdir blockchain-automation-framework
+if NOT DEFINED REPO_BRANCH set "REPO_BRANCH=develop"
+"C:\Program Files\Git\bin\sh.exe" --login -i -c "git checkout %REPO_BRANCH%"
```

Comment:
This should checkout "develop" branch by default and then create a "local" branch. (check the dev_prereq guide)